packages/flutter/lib/src/material/chip.dart

```diff
@@ -1709,6 +1719,20 @@ class _RawChipState extends State<RawChip> with TickerProviderStateMixin<RawChip
     });
   }
 
+  /// Resolve shape border side color, if it's a a MaterialStateColor.
```

Comment:
nit: a a

Reply:
[MaterialStateColor]

Reply:
should this be called getResolvedShape?

Reply:
Good catch, and yes, that sounds like a better name. Will update!